observatory-platform/observatory/platform/utils/test_utils.py

```diff
@@ -653,13 +654,13 @@ def assert_table_content(self, table_id: str, expected_content: List[dict] = Non
         actual_content = None
         try:
             rows = self.bigquery_client.list_rows(table_id)
-            actual_content = [dict(row) for row in rows]
+            actual_content = json.loads(json.dumps([dict(row) for row in rows], default=str))
         except NotFound:
             pass
 
         self.assertIsNotNone(rows)
         if expected_content is not None:
-            for row in expected_content:
+            for row in json.loads(json.dumps(expected_content, default=str)):
```

Comment:
Will it still work without this change, given we're expecting `expected_content: List[dict]` ?

Reply:
Yes, and also with the use-case that I've been testing with (loading a .json from file). The function only requires a List[dict] though, so I figured that it's possible it's handed a table in the same format that it loads the ```actual_content```
```
rows = self.bigquery_client.list_rows(table_id)
actual_content = [dict(row) for row in rows]
```
In this case, the ```actual_content``` would be stringified, but the ```expected_content``` would not be

Reply:
I think the problem is that `dict()` doesn't convert objects to strings and the dates end up as a `FakeDate` object from the BigQuery (not sure why it isn't a datetime), whereas the dates loaded from JSON are strings.